src/widgets/section/SectionList.tsx

```diff
@@ -41,12 +48,105 @@ export default class SectionList extends WidgetBase<SectionListParameters> {
 		);
 	}
 
+	private _getContent() {
```

Comment:
can you just inline this function back into the render, it makes it easier to read.

Reply:
Yeah, meant to do that.